docs/dataset/core-concepts/dataset.md

```diff
@@ -19,9 +19,9 @@ Datapoints are integral components within the dataset structure used for evaluat
 
 - **Exclusive Association with Datasets**: Datapoints are exclusive to the dataset they belong to and are not shared across different datasets. This exclusivity ensures clear demarcation and management of data within specific datasets.
 
-- **Role in Data Ingestion**: Datapoints play a central role in the data ingestion process. They are represented in a DataFrame structure with special treatment for certain columns like *locator* and *text*.
+- **Role in Data Ingestion**: Datapoints play a central role in the data ingestion process. They are represented in a DataFrame structure with special treatment for certain columns like `locator` and `text`.
 
-- **Extension of Data Classes**: Datapoints extend data classes, allowing for flexibility and customization. For instance, they can include annotation objects like *List[BoundingBox]*, and these objects can be further extended as needed.
+- **Extension of Data Classes**: Datapoints extend data classes, allowing for flexibility and customization. For instance, they can include annotation objects like `List[BoundingBox]`, and these objects can be further extended as needed.
```

Comment:
If you put `BoundingBox` in square brackets like this it will link to the corresponding API docs:

```
[`BoundingBox`][kolena.annotation.BoundingBox]
```

<img width="197" alt="Screenshot 2024-01-09 at 9 25 30 PM" src="https://github.com/kolenaIO/kolena/assets/9648886/8ad2b88d-714a-4310-bd15-d54f6b4c313f">